{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:canonical:subiquity:*:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/CanonicalLtd/subiquity/commit/7db70650feaf513d7fb6f1ca07f2d670a0890613",
        "version_end_excluding": {
            "canonical:subiquity": "20.05.2"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "Other"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2020-11932"
        ],
        "CWE": [
            "CWE-532"
        ]
    }
}